CHANGELOG.md

```diff
@@ -6,119 +6,77 @@ This changelog documents user-facing updates (features, enhancements, fixes, and
-
-#### 1.1.5.dev64 (2025-10-02)
-
-- Replaced the `--no-confirm` option with `--yes` in the `modal environment delete` CLI to align with similar interfaces.
-
-#### 1.1.5.dev63 (2025-10-02)
-
-- Added `modal shell` support for connecting to a running Sandbox (`modal shell sb-id`).
-
-#### 1.1.5.dev56 (2025-09-29)
-
-- Added `modal.experimental.image_delete()` to allow deleting Images (e.g. Sandbox FS snapshot images).
-
-#### 1.1.5.dev50 (2025-09-26)
-
-- Fixed bug where large outputs in debug shells would sometimes freeze until key-press.
-
-#### 1.1.5.dev49 (2025-09-26)
-
-- Arbitrary key-value metadata can now be attached to Apps by setting `modal.App(tags={...})`. The tags can be useful for tracking information that may be relevant to your organization, such as the team that owns the App. We'll support the inclusion of tags in some forthcoming APIs related to cost insights.
-
-#### 1.1.5.dev45 (2025-09-25)
-
-- [Internal] Adds support for calling into deployed functions using a new cbor based serialization format used by beta versions of modal-ts and modal-go
-
-#### 1.1.5.dev44 (2025-09-25)
-
-- `modal.Dict.pop()` now accepts an optional `default` parameter, matching Python's `dict.pop()` behavior.
-
-#### 1.1.5.dev40 (2025-09-24)
+Tags can also be added to an active App via the new `modal.App.set_tags()` method, and current tags can be retrieved with the new `modal.App.get_tags()`.
 
-- Hide the CLI spinner in interactive mode, so `modal run --interactive` now works better with breakpoints in local entrypoint functions.
+This release also introduces a new API for generating a tabular billing report: `modal.billing.workspace_billing_report()`. The billing API will report the cost incurred by each App, aggregated over time intervals (currently supporting a daily or hourly resolution). The report can optionally include App tags, allowing you to perform cost allocation using your own organizational schema.
 
-#### 1.1.5.dev34 (2025-09-19)
+Note that the initial release of the billing API is a private beta. Please get in touch to discuss access.
 
-- `Cls.with_options` supports `CloudBucketMount` in `volumes`.
+This release also includes some internal changes to Function input/output serialization. These changes will provide better support for calling into Modal Functions from our `modal-js` and `modal-go` SDKs. Versions 0.4 or later of `modal-js` and `modal-go` will only be able to invoke Functions in Apps deployed with version 1.2 or later of the Python SDK.
 
-```python
-cloud_bucket = modal.CloudBucketMount("my-bucket", secret=aws_secret)
-MyAppCloud = modal.Cls.from_name("my-app", "MyApp").with_options(
-  volumes={"/mnt": cloud_bucket}
-)
-```
+Other new features and improvements:
 
-#### 1.1.5.dev32 (2025-09-19)
+- The new `modal.Sandbox.create_connect_token()` method facilitates authentication for making HTTP / Websocket requests to a server running in a Sandbox:
```

Comment:
Should we mention this considering it's "alpha" and hidden from reference docs at the moment?